main.py

```diff
@@ -205,27 +219,27 @@ def train(train_loader, model, criterion, optimizer, epoch, normalizer):
     model.train()
 
     end = time.time()
-    for i, (input, target, _) in enumerate(train_loader):
+    for i, (_input, target, _) in enumerate(train_loader):
```

Comment:
Change the `_input` back to `input` for naming conventions.

Reply:
According to my linter, the name "input" shades the builtin "input", which is used for reading input from keyboard, therefore it suggests the change of the name.

Reply:
I understand your concern. But `_input` usually means internal variables which is not appropriate in this case. Let's keep the name `input` for readability.